test/tinytest.c

```diff
@@ -350,7 +351,7 @@ testcase_run_one(const struct testgroup_t *group,
 		if (opt_verbosity>0 && !opt_forked)
 			puts("SKIPPED");
 	} else {
-		if (!opt_forked)
+		if (!opt_forked && (testcase->flags & TT_RETRIABLE) && !test_attempts)
```

Comment:
Thank you for the fix!
However it is not a full fix, since assertion errors will be printed anyway, and this will confuse users anyway (that's why I did not bother about extra FAILED message)

Reply:
```suggestion
		if (!opt_forked && test_attempts == 1)
```

Reply:
> Thank you for the fix! However it is not a full fix, since assertion errors will be printed anyway, and this will confuse users anyway (that's why I did not bother about extra FAILED message)

Hi,

I hesitated to also remove the assertion errors, but I think that can be useful to understand why a test has been retried.

Best regards,
Thomas

Reply:
Sounds questionable, but OK.